backend/services/users/src/routes.py

```diff
@@ -0,0 +1,46 @@
+from src import app
+from flask import request, jsonify
+from src import db
+
+
+@app.route("/")
+def index():
+    return jsonify(
+        status=True,
+        message="Hello Users of " + db.name
+    )
+
+
+@app.route("/users")
+def users():
+    print(db.values)
+    userList = db.users.find()
+    
+    user = {}
+    data = []
+
+    for usr in userList:
+        user = {
+            'name' : usr['name'],
+            'privateKey' : usr['privateKey']
```

Comment:
All of the key-related stuff, including all DB stuff, is gonna be scratched due to the direction shift. Sorry Akshay!

Reply:
Yeah, no problem. I had an early night yesterday and woke up to a bunch of messages, I'm trying to catch up but would you guys mind filling me in 9n the design direction during  the next meet as well?